synapse/module_api/__init__.py

```diff
@@ -784,6 +786,8 @@ def invalidate_access_token(
     ) -> Generator["defer.Deferred[Any]", Any, None]:
         """Invalidate an access token for a user
 
+        Can only be called from the main process.
```

Comment:
Is this a new requirement, or are we just making this explicit?

Reply:
Just making it explicit.